spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-loesningsbeskrivelse.md

```diff
@@ -9,7 +9,14 @@ Kjøpekontrakt eller informasjon fra kjøpekontrakten skal kunne oversendes fra
 
 En bank kan sende forespørsel om kjøpekontrakt til en megler basert på kjøpers fødselsnummer (11 siffer). Megler vil besvare forespørselen med en forsendelse som inneholder strukturerte data, samt en signert versjon av den fulle kontrakten. Dersom den faktiske kjøpekontrakten ikke er signert, skal kun den strukturerte delen returneres. Dersom forespørselen ikke kan besvares, vil banken få en feilmelding i retur som beskriver hvorfor megler ikke kan besvare forespørselen.
```

Comment:
Her bør vi kanskje notere noe om at kjøpers fødselsnr potensielt kan treffe på flere oppdrag for den aktuelle meglingeren og at en liste med 1 eller flere kjøpekontrakter sendes tilbake til banken? I de fleste tilfeller vil det nok bare være 1 oppdrag som matcher forøvrig.

Reply:
Lagt til setning på slutten av avsnittet: I noen tilfeller kan kjøpers fødselsnummer treffe på flere oppdrag for den aktuelle megleren, og en liste med 1 eller flere kjøpekontrakter sendes da til banken.